modules/hiredis/1.3.0/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+  - debian11
+  - ubuntu2404
+  - macos
+  - windows
+  bazel:
+  - 8.x
+  - 7.x
+tasks:
+  validate:
+    name: Validate targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - '//:hiredis'
```

Comment:
Those targets should be pre-fixed with `@hiredis` because it's testing this module as a dependency.

Reply:
See https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#anonymous-module-test